src/main/java/pages/ReportedByMeIssuesPage.java

```diff
@@ -0,0 +1,61 @@
+package pages;
+
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.open;
+
+public class ReportedByMeIssuesPage {
+    public static String REPORTED_BY_ME_ISSUES = "http://jira.hillel.it:8080/browse/SUR01-15?filter=-2";
+    private static SelenideElement element = null;
+    private static String xpathReportedByMeIssue = "";
+    private static String xpathMoreButton = "//span[@class='dropdown-text'][contains(text(),'More')]";
+    private static String xpathDeleteButton = "//span[@class='trigger-label'][contains(text(),'Delete')]";
+    private static String idDeleteButtonInDeleteWindow = "delete-issue-submit";
+
+    public static void deleteIssueReportedByMe(String summary) {
+        open(REPORTED_BY_ME_ISSUES);
+        ReportedByMeIssuesPage.xpathReportedByMeIssue = "//span[@class='issue-link-summary'][contains(text(),'" + summary +"')]";
```

Comment:
Please, update this code with String.format()
https://dzone.com/articles/java-string-format-examples